packages/firestore/test/integration/remote/stream.test.ts

```diff
@@ -148,6 +156,14 @@ describe('Watch Stream', () => {
       });
     });
   });
+
+  it('gets connected event before first message', () => {
```

Comment:
I find the title of this a little misleading because the test doesn't appear to test for any messages. Or does it and I'm just not seeing how?

Reply:
The `awaitCallback()` will fail if some other callback occurs (such as a message). The test doesn't care about if the "message" callback occurs, only that "connected" occurs before it. Does that clarify?